app/views/dashboard/index.html.slim

```diff
@@ -1,7 +1,7 @@
 .alert.alert-warning.unsupported-alert role="alert" hidden="true"
   i.fa.fa-4x.pull-left aria-hidden="true"
   span
-    | A supported deployment of SUSE CaaS Platform requires&nbsp;
+    | A supported deployment of #{Rails.root.join("PRODUCT").read} requires&nbsp;
```

Comment:
To avoid reading the file every time it would be nice to cache the info and extract it to a helper method.

Reply:
updated